src/coreclr/jit/codegencommon.cpp

```diff
@@ -9335,10 +9346,10 @@ void CodeGen::genPoisonFrame(regMaskTP regLiveIn)
 //
 void CodeGen::genBitCast(var_types targetType, regNumber targetReg, var_types srcType, regNumber srcReg)
 {
-    const bool srcFltReg = varTypeUsesFloatReg(srcType) || varTypeIsSIMD(srcType);
+    const bool srcFltReg = varTypeUsesFloatReg(srcType);
```

Comment:
`varTypeIsSIMD()` does seems redundant when used with `varTypeUsesFloatReg()`, not sure why it was added in first place.